pkg/sdk/logging/api/v1beta1/syslogng_output_types.go

```diff
@@ -45,6 +45,7 @@ type SyslogNGOutputSpec struct {
 	SplunkHEC       *output.SplunkHECOutput       `json:"splunk_hec_event,omitempty" syslog-ng:"dest-drv"`
 	Loki            *output.LokiOutput            `json:"loki,omitempty" syslog-ng:"dest-drv"`
 	S3              *output.S3Output              `json:"s3,omitempty" syslog-ng:"dest-drv"`
+	Openobserve     *output.OpenobserveOutput     `json:"openobserve-log,omitempty" syslog-ng:"dest-drv"`
```

Comment:
can't we use openobserve as the key? I would rather name this openobserve and if we happen to want to send something else (like metrics) then we can name it openobserver-metrics, but I don't see that happening at the moment

Reply:
Renamed it accordingly.